ashpd-demo/build-aux/com.belmoussaoui.ashpd.demo.Devel.json

```diff
@@ -14,6 +14,7 @@
         "--socket=wayland",
         "--device=dri",
         "--share=network",
+        "--own-name=com.belmoussaoui.ashpd.demo",
```

Comment:
why?

Reply:
because the app-id in the manifest doesn't match and I get : 
```
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

And it's fatal.

Reply:
I run with fenv

Reply:
then fenv is somehow ignoring the build-options, the app sets that as an app id for devel builds

Reply:
it disallow running with release profile too.

Reply:
I am not really a fan of this addition but if it makes your life easier, sure

Reply:
removed for now.